docker/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  web-server:
+    container_name: web-server
+    build: ./web-server
+    ports: 
+      - "9000:80"
+  core:
+    container_name: core
+    build: ./core
+
+  postgres:
+    image: postgres 
+    container_name: postgres
+    ports: 
+      - "5432:5432"
+    environment:
+      - POSTGRES_DB=sourcesdb 
+      - POSTGRES_USER=postgres 
+      - POSTGRES_PASSWORD=postgres
+    volumes:
+      - /Users/yan/Desktop/postshaer:/var/lib/postgresql
```

Comment:
`/Users/yan/Desktop/postshaer` can't be used in production. Is there a way to provide the path to `docker-compose` when running it, without hardcoding it in the file?

Reply:
Another my fault - forget to delete compose, because we don't use compose.